docs/commands/push.md

```diff
@@ -77,16 +77,17 @@ openapi push [-u] [--run-id id] <path/to/definition.yaml> <@organization-id/api-
---branch, -b    | string  | The branch where your API definition will be pushed or upserted. Default value is `main`.  |
---help       | boolean | Help output for the command.  |
---run-id       | string  | Specify the ID of the CI job that the current push will be associated with. See [the Run ID section](#run-id) for more information.  |
---skip-decorator | [string] | Ignore one or more decorators. See the [Skip decorator section](#skip-decorator) for usage examples.
---upsert, -u | boolean | Upsert an API to the API registry. See [the Upsert an API with push section](#upsert-an-api-with-push) for more information.  |
---version     | boolean | Show version number.  |
---region,-r    | string | Specify which region to use when logging in. Supported values: `us`, `eu`. The `eu` region is limited to enterprise customers. Default value is `us`. Read more about [configuring the region](../configuration/index.mdx#region).
+-----------------|:---------:|:------------:|
+entrypoint       | string    | Optional. The API definition that you want to push to the Redocly API registry. Provide it as a path to the root API definition file (or as an alias from `apiDefinitions` if using the legacy configuration file). See [Set options explicitly](#set-options-explicitly) for more information.  |
+destination      | string    | Required. The location in the API registry where you want to push or upsert your API definition. Provide it in the following format: `@organization-id/api-name@api-version` or `api-name@api-version`if organization ID is already defined in the configuration file. See [the Destination section](#destination) for more information.  |
+--branch, -b    | string  | Optional. The branch where your API definition will be pushed or upserted. Default value is `main`.  |
+--help       | boolean | Optional. Help output for the command.  |
+--run-id       | string  | Optional. Specify the ID of the CI job that the current push will be associated with. See [the Run ID section](#run-id) for more information.  |
+--skip-decorator | [string] | Optional. Ignore one or more decorators. See the [Skip decorator section](#skip-decorator) for usage examples.
+--upsert, -u | boolean | Optional. Upsert an API to the API registry. See [the Upsert an API with push section](#upsert-an-api-with-push) for more information.  |
+--version     | boolean | Optional. Show version number.  |
+--region,-r    | string | Optional. Specify which region to use when logging in. Supported values: `us`, `eu`. Default value is `us`. Read more about [configuring the region](../configuration/configuration-file.mdx#region).
+--public       | boolean | Optional. Make API definition public accessible from API Registry. Read more about [public option](#public). |
```

Comment:
This table was revised. This commit undoes all of the revisions I made. Now, this table is poorly formatted and the content is bad. The standard is that all options are optional unless stated otherwise. This is likely the result of a bad conflict resolution.

Reply:
Yes, it was after conflict resolving, thx, fixed)